modules/ROOT/pages/api-groups-sla-tiers.adoc

```diff
@@ -0,0 +1,46 @@
+= SLA Tiers For API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can specify service-level agreement (SLA) tiers to API Groups instances to limit the number of requests an application (user) can make to the APIs grouped in an API Group. SLAs enable you to protect your API Group instances and help manage network traffic to your APIs.
+
+The SLA tier definition also determines whether access to the API Groups instance requires your approval. You can configure manual or automatic approvals to requests.
+
+== SLA Types
+
+SLA tiers are of two types: 
+
+* Default SLAs
+* Individual API SLAs
+
+== Default 
+
+The default limit is set by the Group Administrator and applies to all API instances. If you add a new API instance to an API group instance, the default limit SLA configured at the group instance level applies, unless an individual API SLA tier set by the Group Administrator. 
```

Comment:
The whole tier is defined by a group administrator

Reply:
Done! Thank!